jumpscale/sals/zos/kubernetes.py

```diff
@@ -71,6 +73,7 @@ def add_worker(
         master_ip: str,
         ssh_keys: List[str],
         pool_id: int,
+        public_ip: int
```

Comment:
we should call it `public_ip_wid` to keep consistency and give it default value to None or do we require public ip ?